bin/bench-tx/bench-tx.json

```diff
@@ -1,21 +1,23 @@
 {
   "simple": {
-    "prologue": 4503,
-    "notes_processing": 2235,
+    "prologue": 4167,
+    "notes_processing": 2540,
     "note_execution": {
-      "0xccc0bb4ef7e27c1fd6ada71b87574e1b48f996b328b5f5d80eb526e6c288e0a6": 755,
-      "0xd6936c529c79d21ddc287db672f07b150b8c06cd8b2b8d9c2c66b1bedb4a67fd": 1439
+      "0x7fe47b84889353de93aea4bfeaa8da5da0036ea7a84dfc5ff5765ec51440896f": 1266,
+      "0xbf1c92f4f0b35b51d3f3e8f7ba19abf7718a7e087627df0d17987af8a1cb872a": 1233
```

Comment:
Any idea why the cycle counts changed like this (almost doubling for one note, and decreasing by 200 cycles for another)?

Reply:
Probably also related to this: https://github.com/0xMiden/miden-base/pull/1654#discussion_r2241605103.

One note previously created one output note while the other created two output notes. Now they don't create output notes, so I assume that's where it comes from.

Reply:
Yes, let's create an issue to add a benchmark for note creation.